playbooks/openstack/inventory.py

```diff
@@ -76,6 +76,16 @@ def build_inventory():
 
     inventory['_meta'] = {'hostvars': {}}
 
+    # cinder volumes used for docker storage
+    docker_storage_volume_mountpoints = {}
+    for volume in cloud.list_volumes():
+        if 'purpose' in volume.metadata and volume.metadata.purpose == "openshift_docker_storage":
```

Comment:
This could be shortened to:

    if volume.metadata.get('purpose') == "openshift_docker_storage":

(the `get` method on dicts returns `None` if empty or you can specify a default value as the second argument)